tfjs-backend-wasm/src/cc/kernels/BitwiseAnd.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2023 Google LLC.
```

Comment:
Can you add a bazel target following other kernels to https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-wasm/src/cc/BUILD.bazel ?
Although our build rule is somehow broken right now which includes all c files when compiling, maintaining a valid BUILD file will make it easier to fix this issue in the future.